python/.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Was this added by accident or is it needed?

Reply:
Oops, that was by accident. I used a notebook to test the rest catalog and go over some of the endpoints.